src/mcp/tools/appdistribution/tests.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from "zod";
+import { ApplicationIdSchema } from "../../../crashlytics/filters";
+import { distribute, Distribution } from "../../../appdistribution/distribution";
+import { tool } from "../../tool";
+import { mcpError, toContent } from "../../util";
+import {
+  parseIntoStringArray,
+  parseTestDevices,
+  toAppName,
+} from "../../../appdistribution/options-parser-util";
+
+export const run_tests = tool(
+  {
+    name: "run_test",
+    description: `Upload an APK and run an existing test against it.`,
+    inputSchema: z.object({
+      appId: ApplicationIdSchema,
+      releaseBinaryFile: z.string().describe("Path to the binary release (APK)."),
+      testDevices: z.string().describe(
+        `Semicolon-separated list of devices to run automated tests on, in the format
+          'model=<model-id>,version=<os-version-id>,locale=<locale>,orientation=<orientation>'. Run 'gcloud firebase test android|ios models list' to see
```

Comment:
Since this is Android-only right now (incl. the fact that "APK" is referenced in the description) I think we can just use "android" instead of "android|ios".